source/js/buyers-guide/components/social-share/social-share.jsx

```diff
@@ -1,33 +1,49 @@
 import React from 'react';
+import ReactGA from 'react-ga';
+import DNT from '../../dnt.js';
 
 const SocialShareLink = (props) => {
   let classes = `social-button`;
   let srLabel = ``;
   let link = `PrivacyNotINcluded.org`;
```

Comment:
Unrelated to this PR, but...  `PrivacyNotIncluded.org`?

Reply:
might as well fix that here I guess